.gitignore

```diff
@@ -1,15 +1,261 @@
-# build and binary files
```

Comment:
Sure, I can revert this, no problem. I actually just needed to exclude a single folder. I'm on Windows using a combination of VS and VSCode